walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -121,6 +124,71 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void everyAttendeeIsConsideredWithOptFullDayEvent() {
```

Comment:
I think these tests should be renamed as such <action>_<givenCertainInput>_<returnsExpctedOutcome>. For this example, supposed we rename the "query" function to be "findAllAvailableTimeSlots", we potentially do "findAllAvailableTimeSlots_withOptionalAttendee_returnsTimeSlotsForRequiredAttendees".